contracts/cwd-core/src/msg.rs

```diff
@@ -212,9 +211,18 @@ pub enum QueryMsg {
         start_after: Option<String>,
         limit: Option<u32>,
     },
-    /// Implements the DAO Star standard: https://daostar.one/EIP
+    /// Implements the DAO Star standard: <https://daostar.one/EIP>
     #[returns(crate::query::DaoURIResponse)]
     DaoURI {},
+    /// Returns the voting power for an address at a given height.
+    #[returns(cwd_interface::voting::VotingPowerAtHeightResponse)]
+    VotingPowerAtHeight {
```

Comment:
hmm maybe we could have a `core_module_query` macro as well? feels a little odd to have some modules have these as macros and some not

Reply:
we totally could just return `env.contract.address` from a `Dao {}` query and be voting module interface compliant.

con: weird looking
pro: consistent, modules can actually treat this like a full blown voting module.

wdyt?

Reply:
I thought the core module is unique in that there would likely only be one implementation verses voting or proposal modules where folks will constantly be writing custom ones.

For the sake of clarity, I think it's fine to not use macros here. :shrug:

Reply:
> we totally could just return `env.contract.address` from a `Dao {}` query and be voting module interface compliant.
> 
> con: weird looking pro: consistent, modules can actually treat this like a full blown voting module.
> 
> wdyt?

Don't have strong opinions here... do think it's a little weird looking... could do that though. Might be cool to use a DAO for voting power... though I think we could do this without the `Dao { }` query?

Reply:
> we totally could just return `env.contract.address` from a `Dao {}` query and be voting module interface compliant.
> 
> con: weird looking pro: consistent, modules can actually treat this like a full blown voting module.
> 
> wdyt?

Don't think we want this since it won't behave correctly. The `Dao { }` query is supposed to return the DAO the voting module belongs to. If it's hard coded to `env.contract.address` and another DAO was using that DAO as a voting module, it would return the incorrect value. Not sure we want to encourage this anyway. :joy:

Leaving as is.

Reply:
> and another DAO was using that DAO as a voting module

this is a genius point